doc/nrf/ug_thingy53.rst

```diff
@@ -295,60 +272,158 @@ You can program the precompiled firmware image using one of the following ways:
+This ensures that the sample includes the MCUboot bootloader and that an MCUboot-compatible image is generated when the sample is built.
 
-In addition to the configurations mentioned above, an application compatible with Thingy:53 is expected to enable the following features:
+The MCUboot bootloader supports serial recovery and a custom command to erase the settings storage partition.
+Erasing the settings partition is needed to ensure that an application is not booted with incompatible content loaded from the settings partition.
 
-* USB CDC ACM as backend for logger.
-  The logs are provided using USB CDC ACM to allow accessing them without additional hardware.
+To enable the MCUboot bootloader, enable the following Kconfig options:
 
-  Most of the Thingy:53-compatible applications and samples use only a single instance of USB CDC ACM that works as logger's backend.
-  No other USB classes are used.
-  These samples can share common USB product name, vendor ID, and product ID.
-  If a sample supports additional USB classes or more than one instance of USB CDC ACM, it must use dedicated product name, vendor ID, and product ID.
+* :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT`
+* :kconfig:option:`CONFIG_IMG_MANAGER`
+* :kconfig:option:`CONFIG_MCUBOOT_IMG_MANAGER`
+* :kconfig:option:`CONFIG_IMG_ERASE_PROGRESSIVELY`
 
-* DFU over-the-air through Simple Management Protocol over Bluetooth.
-  The application acts as GATT server and allows the connected Bluetooth Central to perform the firmware update for both application and network core.
-  The Bluetooth configuration is updated to allow quick DFU data transfer.
-  The application supports Simple Management Protocol (SMP) handlers related to:
+In addition, set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER` option to ``2`` and :kconfig:option:`CONFIG_MCUBOOT_IMAGE_VERSION` to ``"1.0.0+0"``.
 
-  * Image management
-  * Operating System (OS) management -- used to reboot device after firmware upload is complete.
-  * Erasing settings partition -- needed to ensure that a new application would not be booted with incompatible content in the settings partition that was written by the previous application.
+See :file:`thingy53_nrf5430_cpuapp.conf` in a compatible sample for how this is done.
 
-.. tip::
-   To use your application with Thingy:53 preprogrammed bootloader, you only must set proper configuration of Partition Manager and MCUboot.
-   Other mentioned features are nice to have, but they are not mandatory.
+.. _thingy53_app_usb:
 
-   All of the Thingy:53-compatible applications in the |NCS| also support the mentioned additional features.
-   Refer to the source code of the samples and applications for examples of implementation.
+USB
+===
+
+An application compatible with the Nordic Thingy:53 is expected to enable USB CDC ACM as back-end for logging.
```

Comment:
back end or back-end (or even backend)

Reply:
With a quick search, "backend" is the most common in our documentation.